src/dev/amdgpu/amdgpu_gfx.cc

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
I am not an expert at reading the legalese at the top, but just verifying you made sure this doesn't include whatever problematic phrases caused trouble a while back?

Reply:
Yea it's the standard 3 clause BSD... the problematic line is not there

Reply:
thanks both